core/src/main/java/org/arquillian/smart/testing/hub/storage/local/LocalChangeStorage.java

```diff
@@ -0,0 +1,75 @@
+package org.arquillian.smart.testing.hub.storage.local;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.arquillian.smart.testing.hub.storage.ChangeStorage;
+import org.arquillian.smart.testing.scm.Change;
+
+public class LocalChangeStorage implements ChangeStorage {
+
+    private static final String SMART_TESTING_SCM_CHANGES = "smart-testing-scm-changes";
+
+    private final String currentDirectory;
+
+    // Used by SPI
+    @SuppressWarnings("unused")
+    public LocalChangeStorage() {
+        this.currentDirectory = ".";
+    }
+
+    public LocalChangeStorage(String currentDirectory) {
+        this.currentDirectory = currentDirectory;
+    }
+
+    @Override
+    public void store(Collection<Change> changes) {
+        StringBuffer fileContent = new StringBuffer();
```

Comment:
Stupid thing but you can use `StringBuilder` it is fastest to use rather than `StringBuffer`